src/auth-service/bin/test/ut_index.js

```diff
@@ -1,12 +1,33 @@
 require("module-alias/register");
 const { expect } = require("chai");
 const sinon = require("sinon");
+const proxyquire = require("proxyquire");
 
 // Import the functions to be tested
-const kafkaConsumer = require("@bin/kafka-consumer");
-const createServer = require("@bin/server");
+// const kafkaConsumer = require("@bin/kafka-consumer");
+// const createServer = require("@bin/server");
 const main = require("@bin/index");
 
+const kafkaConsumer = proxyquire("@bin/kafka-consumer", {
```

Comment:
Using `proxyrequire `to navigate `import errors` for unit tests. @BenjaminSsempala